turbinia/workers/analysis/yara.py

```diff
@@ -52,6 +76,9 @@ def run(self, evidence, result):
     Returns:
         TurbiniaTaskResult object.
     """
+    # Let's update the Yara rules
+    self.update_rules()
```

Comment:
Given this analysis job runs very frequently, and could be executed hundreds of workers in a short amount of time, could we run into rate limiting issues from github?

Reply:
Good point. I don't think it will run hundreds of times. The Yara Analyser supports a limited amount of Evidence types and only runs on a container or partition and will only run 1 time for an Evidence type. 

So let's say you have a cloud disk, it will have a couple of partitions (we already only analyse the big partition afaik) so the Yara rules will update 1 time. When that partition has 5 containers, each container will be run through the analyser as well, so 5 more times.

I was considering putting in the Worker startup code but that would indeed kick it off too many times.

Having said that I am only aware of rate limits for the Github API (api rest endpoint) and not for git cli usage.

Reply:
Makes sense. SGTM.